src/dependency-manager/src/component-config/v1.ts

```diff
@@ -84,7 +84,7 @@ export const transformInterfaces = function (input?: Dictionary<string | Diction
         host = `\${{ ${matches[1]}.host }}`;
         port = `\${{ ${matches[1]}.port }}`;
         protocol = `\${{ ${matches[1]}.protocol }}`;
-        url = `\${{ ${matches[1]}.protocol }}://\${{ ${matches[1]}.host }}:\${{ ${matches[1]}.port }}`;
+        url = `\${{ ${matches[1]}.url }}`;
```

Comment:
Worried about this change. Do we have any tests for host overrides? 

ex:
```yml
# architect.yml
name: examples/react-app
services:
  api:
    interfaces:
      main: 81
  app:
    environment:
      API_ADDR: ${{ services.api.interfaces.main.url }}
```

```yml
# environment.yml
components:
  examples/react-app:
    extends: latest
    services:
      api:
        interfaces:
          main:
            host: google.com
            port: 443
```

`expect(app_env.API_ADDR).to.eq('https://google.com')`